src/components/Tag.tsx

```diff
@@ -46,21 +49,27 @@ const StyledIcon = styled(Icon)`
   margin-left: 1em;
 `
 
-const Tag = ({
+export interface IProps {
+  name: string
+  key: string
```

Comment:
There is no need to specify `key` in this component. `key` is used on the consumer, when you need to do a `.map` or a loop but inside the component, you don't need to pass that as a prop.

Reply:
Make sense, thank you.